python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,22 +365,24 @@ def __get__(self, obj, cls):
             return cls._schema[name]
 
         if self._getter_name:
-            return self._bind_method(obj, self._getter_name)()
+            returnable = self._bind_method(obj, self._getter_name)()
+            if isinstance(returnable, GangaObject):
+                returnable._setParent(self)
+            return returnable
```

Comment:
This leads me to suspect that a job is being created without knowledge of it's parent using a getter method then? This confuses me as to why this would be a good idea to have this happen.